homeassistant/components/tuya/number.py

```diff
@@ -70,6 +70,18 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    "dj": (
+        NumberEntityDescription(
+            key=DPCode.COUNTDOWN_1,
+            name="Timer",
+            icon="mdi:timer",
+            entity_category=EntityCategory.CONFIG,
+            max_value="86400",
+            min_value="0",
+            native_unit_of_measurement=TIME_SECONDS,
+            native_step="1",
+        ),
```

Comment:
This entity will count-down (as in, actively change state during the count-down is going on), which is not allowed.